repository.eselect.in

```diff
@@ -279,6 +323,9 @@ describe_enable_options() {
 do_enable() {
 	[[ ${#} -gt 0 ]] || die -q "no repositories specified"
 
+	# TODO: make command line option. one of auto, ask, false.
+	local -x add_recursive="auto"
```

Comment:
Could this behavior be configured in `/etc/eselect/repository.conf` as well, like Anna mentioned?
Would it be easier? Or the other way around?

Reply:
Sure. My idea was that this gets a default which can be overidden by the config file and an explicit command line argument. That is pretty standard in such cases.